docs/src/pages/components/button.mdx

```diff
@@ -398,6 +398,14 @@ The `BackButton` component is a preset based on the `Button` component.
 
 <PropsTable of="BackButton" />
 
+# Buttons as links
+
+Buttons can be used as links by using the optional `is` property with a value such as `ReactRouterLink` from [react-router](https://github.com/ReactTraining/react-router), or `a`.
```

Comment:
I think it's just `Link` from `react-router`, not `ReactRouterLink`.

Reply:
Could you also add ", or `a` tag." at the end?

Reply:
For sure, thanks!

Reply:
Added.